docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md

```diff
@@ -166,248 +166,170 @@ You need to perform the following steps:
+  quay.io/yugabyte/thirdparty-deps   latest         479490b66ad9   4 weeks ago    665MB
+  prom/prometheus                    v2.41.0        932c2dbe7d3e   6 weeks ago    231MB
+  postgres                           14.4           e09e90144645   6 months ago   376MB
+  ```
+
+  ```sh
+  docker tag 230d619f5c20 gcr.io/dataengineeringdemos/yugabyte/yugaware:2.16.1.0-b50
+  docker tag 479490b66ad9 gcr.io/dataengineeringdemos/yugabyte/thirdparty-deps:latest
+  docker tag 63e2b5caa055 gcr.io/dataengineeringdemos/yugabyte/nginxinc/nginx-unprivileged:1.23.3
+  docker tag e09e90144645 gcr.io/dataengineeringdemos/yugabyte/postgres:14.4
+  docker tag e4e13bbe3446 gcr.io/dataengineeringdemos/tianon/postgres-upgrade:11-to-14
+  docker tag 932c2dbe7d3e gcr.io/dataengineeringdemos/prom/prometheus:v2.41.0
+  ```
 
 1. Push images to the private container registry, as follows:
 
-   ```sh
-   docker push a04fef023c7c 
-   docker push 721453480a0f 
-   docker push 5a9061639d0a 
-   docker push 5f1485c70c9a
-   docker push cc866859f8df 
-   docker push caef6233eac4 
-   ```
-
-   ![img](/images/yp/docker-image.png)
-
-1. Modify the Helm chart `values.yaml` file. You can map your private internal repository URI to `commonRegistry` and use the folder or `project/image_name` and tags similar to the following:
-
-   ```properties
-   image:
-      commonRegistry: "**gcr.io/dataengineeringdemos**"
-      repository: **""**
-      tag: **2.5.1.0-b153**
-      pullPolicy: IfNotPresent
-      pullSecret: yugabyte-k8s-pull-secret
-      thirdparty-deps:
-      		registry: /yugabyte/thhirdparty-deps
-   			tag: **latest**
-   			name: **yugabyte/thirdparty-deps**
-      postgres:
-   			registry: "yugabyte/postgres"
-   			tag: 11.5
-   			name: **postgres**
-      prometheus:
-   			registry: "prom/prometheus"
-   			tag: **v2.2.1**
-   			name: **prom/prometheus**
-      nginx:
-   			registry: "yugabyte/nginx"
-   			tag: **1.17.4**
-   			name: nginx
-      dnsmasq:
-   			registry: "janeczku/go-dnsmasq/"
-   			tag: **release-1.0.7**
-   			name: **janeczku/go-dnsmasq
-   ```
-
-1. Install Helm chart or specify the container registry in YugabyteDB Anywhere cloud provider, as follows:
-
-   ```sh
-   helm install yugaware **.** -f values.yaml
-   ```
+  ```sh
+  docker push gcr.io/dataengineeringdemos/yugabyte/yugaware:2.16.1.0-b50
+  docker push gcr.io/dataengineeringdemos/yugabyte/thirdparty-deps:latest
+  docker push gcr.io/dataengineeringdemos/yugabyte/nginx:1.23.3
+  docker push gcr.io/dataengineeringdemos/yugabyte/postgres:14.4
+  docker push gcr.io/dataengineeringdemos/janeczku/tianon/postgres-upgrade:11-to-14
+  docker push gcr.io/dataengineeringdemos/prom/prometheus:v2.41.0
+  ```
+
+  ![img](/images/yp/docker-image.png)
+
+1. Modify the Helm chart `values.yaml` file. You can map your private internal repository `gcr.io/dataengineeringdemos` URI to `commonRegistry`.
+
+  ```properties
+  image:
+    commonRegistry: "gcr.io/dataengineeringdemos"
```

Comment:
It is actually possible to just set the `image.commonRegistry` to get images from a different registry. We don't need to add anything else to the values apart from pullSecret (that's because the target registry can also be private).

The only thing one needs to ensure is images are tagged correct in the private registry. 
Do `helm template . --set image.commonRegistry=gcr.io/myreg` to see how the image tags are generated for all the containers. And then we can make changes in the rest of the instructions accordingly.